ql/src/test/queries/clientnegative/materialized_view_no_cbo_rewrite.q

```diff
@@ -1,11 +0,0 @@
-set hive.support.concurrency=true;
-set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
-set hive.strict.checks.cartesian.product=false;
-set hive.materializedview.rewriting=true;
-
-create table cmv_basetable (a int, b varchar(256), c decimal(10,2))
-stored as orc TBLPROPERTIES ('transactional'='true');
-
-insert into cmv_basetable values (1, 'alfred', 10.30),(2, 'bob', 3.14),(2, 'bonnie', 172342.2),(3, 'calvin', 978.76),(3, 'charlie', 9.8);
-
-create materialized view cmv_mat_view as select a, b, c from cmv_basetable sort by a;
```

Comment:
Would this still fail for queries that do not go through CBO due to missing support for some features/constructs? We do not have many of those queries left but maybe we should add the negative test (if it is not already there)

Reply:
Could you please describe this.
I checked locally that If CBO is turned off materialized view creation will fail with `SemanticException: Cannot enable automatic rewriting for materialized view.` No matter what was the query definition.
Should a general negative test be added? like
```
set hive.cbo.enable=false;
create materialized view mat1 as select col from t;
```
I haven't found tests targeting this.

Reply:
Added test described above.